node/cn/tracers/api_test.go

```diff
@@ -171,17 +182,74 @@ func (b *testBackend) StateAtTransaction(ctx context.Context, block *types.Block
 		txContext := blockchain.NewEVMTxContext(msg, block.Header(), b.chainConfig)
 		blockContext := blockchain.NewEVMBlockContext(block.Header(), b.chain, nil)
 		if idx == txIndex {
-			return msg, blockContext, txContext, statedb, nil
+			return msg, blockContext, txContext, statedb, release, nil
 		}
 		vmenv := vm.NewEVM(blockContext, txContext, statedb, b.chainConfig, &vm.Config{Debug: true, EnableInternalTxTracing: true})
 		if _, err := blockchain.ApplyMessage(vmenv, msg); err != nil {
-			return nil, vm.BlockContext{}, vm.TxContext{}, nil, fmt.Errorf("transaction %#x failed: %v", tx.Hash(), err)
+			return nil, vm.BlockContext{}, vm.TxContext{}, nil, nil, fmt.Errorf("transaction %#x failed: %v", tx.Hash(), err)
 		}
 		statedb.Finalise(true, true)
 	}
-	return nil, vm.BlockContext{}, vm.TxContext{}, nil, fmt.Errorf("transaction index %d out of range for block %#x", txIndex, block.Hash())
+	return nil, vm.BlockContext{}, vm.TxContext{}, nil, nil, fmt.Errorf("transaction index %d out of range for block %#x", txIndex, block.Hash())
 }
 
+// TODO-kaia: enable testcode
```

Comment:
what's this?

Reply:
Oh thank you, I forgot to enable this testcode. Let me enable this.